loadtest/loadtest_client.go

```diff
@@ -87,12 +86,14 @@ func NewLoadTestClient(config Config) *LoadTestClient {
 }
 
 func (c *LoadTestClient) SetValidators() {
-	if strings.Contains(c.LoadTestConfig.MessageType, "staking") {
-		resp, err := c.StakingQueryClient.Validators(context.Background(), &stakingtypes.QueryValidatorsRequest{})
-		if err != nil {
-			panic(err)
+	for _, mType := range c.LoadTestConfig.MessageTypes {
+		if mType == Staking {
```

Comment:
we probably need to re-think staking- thoughts on commenting this out or jsut adding a print saying "Staking isn't supported yet"?